tools/packaging/packaging.psm1

```diff
@@ -1710,14 +1713,397 @@ function CreateNugetPlatformFolder
     )
 
     $destPath = New-Item -ItemType Directory -Path (Join-Path $PackageRuntimesFolder "$Platform/lib/$script:netCoreRuntime")
-    $fullPath = Join-Path $PlatformBinPath $file
+    $fullPath = Join-Path $PlatformBinPath $FileName
 
     if (-not(Test-Path $fullPath)) {
         throw "File not found: $fullPath"
     }
 
     Copy-Item -Path $fullPath -Destination $destPath
-    Write-Log "Copied $file to $Platform"
+    Write-Log "Copied $FileName to $Platform at path: $destPath"
+}
+
+<#
+.SYNOPSIS
+Creates a CGManifest file containing package dependencies for specified file.
+
+.PARAMETER FilePath
+File path name of CGManifest file to be created.
+
+.PARAMETER Dependencies
+Array list of dependency tuples:
+[tuple[ [tuple[string, string]], [tuple[string, string]] ] []]
+["Id", "Microsoft.PowerShell.SecretStore"], ["Version", "1.1.1.0"]
+#>
+function New-CGManifest
+{
+    param (
+        [parameter(Mandatory = $true)]
+        [string] $FilePath,
+
+        [parameter(Mandatory = $false)]
+        [tuple[ [tuple[string, string]], [tuple[string, string]] ] []] $Dependencies
```

Comment:
nit: Maybe a comment about the tuple structure

Reply:
This is commented above at line 1734.